runtime/convertValues.go

```diff
@@ -446,7 +446,8 @@ func importValue(inter *interpreter.Interpreter, value cadence.Value, expectedTy
 	case cadence.Bool:
 		return interpreter.BoolValue(v), nil
 	case cadence.String:
-		return interpreter.NewStringValue(string(v)), nil
+		// TODO: meter?
```

Comment:
I feel like this probably does need metering; if we only meter strings that users create from within their smart contracts rather than strings that they import from storage or elsewhere, this creates a weird incentive for people to never create strings in their code and instead load them from outside.

Reply:
👍 

Added in c4c2d5e